src/main/java/com/google/maps/model/Photo.java

```diff
@@ -0,0 +1,28 @@
+package com.google.maps.model;
+
+/**
+ * <p>This describes a photo available with a Search Result.</p>
+ *
+ * <p>Please see <a href="https://developers.google.com/places/web-service/photos">Photos</a> for more details.</p>
+ */
+public class Photo {
```

Comment:
I had a go at doing something along those lines for `nextPageToken` - but it wound up involving given the result object knowledge of the context the original request was made on. 

I feel for API regularity I need to introduce a method on `PlacesApi` that takes a `photoReference` as the argument. However, I still need to modify that request such that GSON isn't used to attempt to marshall raw PNG data. Hmm.